test/test_torch.py

```diff
@@ -10053,7 +10053,7 @@ class MyTwoTensor4(TwoTensor):
 METHOD = 1
 INPLACE_METHOD = 2
 FUNCTIONAL = 4
-DIM_ARG = None
+DIM_ARG: None = None
```

Comment:
Huh, never seen this annotation before. Didn't know that worked (should you use it instead of `type(None)`?)